src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests/Functions/ProcessCompletionOfNextDataSetVersionImportFunctionTests.cs

```diff
@@ -0,0 +1,184 @@
+using GovUk.Education.ExploreEducationStatistics.Common.Tests.Extensions;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Database;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Model.Parquet.Tables;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Functions;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Model;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Services.Interfaces;
+using Microsoft.DurableTask;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Moq;
+using Moq.Protected;
+using static GovUk.Education.ExploreEducationStatistics.Common.Tests.Utils.MockUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Tests.Functions;
+
+public abstract class ProcessCompletionOfNextDataSetVersionImportFunctionTests(
```

Comment:
I think we're missing some tests for `ProcessCompletionOfNextDataSetVersionFunction.CompleteNextDataSetVersionImportProcessing`

Reply:
Haha yup, I have updated ProcessInitialDataSetVersionFunctionTests to be CompleteNextDataSetVersionImportProcessingTests, and testing the correct function method now thanks!